cardano-cli/src/Cardano/CLI/Shelley/Commands.hs

```diff
@@ -84,6 +98,17 @@ data AddressCmd
   | AddressInfo Text (Maybe OutputFile)
   deriving (Eq, Show)
 
+
+renderAddressCmd :: AddressCmd -> Text
+renderAddressCmd cmd =
+  case cmd of
+    (AddressKeyGen _ _ _) -> "address key-gen"
+    (AddressKeyHash _ _) -> "address key-hash"
+    (AddressBuild _ _ _ _) -> "address build"
+    (AddressBuildMultiSig) -> "address build-multisig"
+    (AddressInfo _ _) -> "address info"
+    (AddressBuildScript _ _ _) -> "address build-script"
```

Comment:
I'm hesitant of doing it this way when the strings used in `Cardano.CLI.Shelley.Parsers` could diverge from these.

Reply:
cc @erikd @newhoggy

Reply:
They already diverge somewhat already don't they?  Do you expect the above strings to be embedded in the program description?

Reply:
It seems like these strings are supposed to be the same as the subcommand strings, e.g. `cardano-cli shelley address key-gen`. If we were to make a change to one of those in `Cardano.CLI.Shelley.Parsers`, then we'd need to remember to also make that change here. Maybe it's not a big problem, but I thought it'd be worth mentioning.

Reply:
I had another look and it seems the text is meant to resemble the command used in the CLI not the program description.  And that both should resemble the constructor name, so it's probably fine?

Reply:
`(AddressKeyGen _ _ _) -> ` will break as soon as parameter is added or removed.

`AddressKeyGen {}` wil not break.

Reply:
I somewhat agree.  Which is better depends on whether you want it to break.  Probably `hlint` suggested using `{}`, if we know better, we can suppress with a `HLINT` pragma.  There are examples in many Haskell files.

Reply:
Hlint is correct in this case :D

Reply:
@newhoggy at this point we're only concerned with the name of the command, not necessarily it's arguments so `{}` is fine for now.